integration-tests/src/test/resources/docker-compose.yml

```diff
@@ -3,27 +3,30 @@ version: '3.7'
 services:
   zookeeper:
     image: bitnami/zookeeper:3.5.5
+    container_name: "zookeeper"
```

Comment:
That name is used as a global unique identifier.
The value "zookeeper" is not unique enough and if some other service would use the same, we will get annoying conflicts.

Let's add "tw-tasks" prefix for all containers in this file.